concent_api/delete-unsupported-messages.py

```diff
@@ -0,0 +1,40 @@
+from logging import getLogger
+import os
+import sys
+
+import django
+from django.db import transaction
+
+from common.helpers import deserialize_message
+
+
+sys.path.append('concent_api')
+os.environ['DJANGO_SETTINGS_MODULE'] = "concent_api.settings"
+django.setup()
+
+from core.models import StoredMessage  # noqa E402
+from core.models import Subtask  # noqa E402
+
+logger = getLogger(__name__)
+
+
+def main() -> None:
+    with transaction.atomic(using='control'):
+        for subtask in Subtask.objects.select_for_update().all():
+            try:
+                report_computed_task = deserialize_message(subtask.report_computed_task.data.tobytes())
+                if not isinstance(report_computed_task.size, int):
+                    logger.info(f'Size inside ReportComputedTask message inside subtask with subtask_id: {subtask.subtask_id} is: {report_computed_task.size}')
```

Comment:
Logging each subtask twice makes it harder for the admin to see irregularities in the output. Please log one line per subtask and make it less verbose:

Why not:
``` python
f"Deleting subtask {subtask:5} and. REASON: ReportComputedTask.size not an int: {report_computed_task.size:10}"
```

After changing this please post a sample of the output in a comment here.